cmd/ssh.go

```diff
@@ -284,3 +456,87 @@ func (cmd *SSHCmd) startServices(ctx context.Context, devPodConfig *config.Confi
 		}
 	}
 }
+
+// setupGPGAgent will forward a local gpg-agent into the remote container
+// this works by using cmd/agent/workspace/setup_gpg
+func (cmd *SSHCmd) setupGPGAgent(
+	ctx context.Context,
+	containerClient *ssh.Client,
+	log log.Logger,
+) error {
+	writer := log.ErrorStreamOnly().Writer(logrus.InfoLevel, false)
+	defer writer.Close()
+
+	log.Debugf("gpg: exporting gpg public key from host")
+
+	// Read the user's public keys and ownertrust from GPG.
+	// These commands are executed LOCALLY, the output will be imported by the remote gpg
+	pubKeyExport, err := exec.Command("gpg", "--armor", "--export").Output()
```

Comment:
why won't we move those commands (this, and ones below) to gpg package? 

I feel more comfortable with a "client" calling `exec.Command` underneath instead of calling in directly in hear.

Reply:
Done :+1: